diff --git a/python/setup.py b/python/setup.py
index f625bae8c..aaa1135a4 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -155,6 +155,12 @@ def config_cython():
         print("WARNING: Cython is not installed, will compile without cython module")
         return []
 
+from setuptools.command.install import install
+class InstallPlatlib(install):
+    def finalize_options(self):
+        install.finalize_options(self)
+        self.install_lib = self.install_platlib
+
 
 class BinaryDistribution(Distribution):
     def has_ext_modules(self):
@@ -233,6 +239,7 @@ setup(
     package_data={"tvm": get_package_data_files()},
     distclass=BinaryDistribution,
     ext_modules=config_cython(),
+    cmdclass={'install': InstallPlatlib},
     **setup_kwargs,
 )
 
